Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM ubuntu:xenial
+FROM ubuntu:18.04
 RUN apt-get update -qq && apt-get upgrade -y
 
-RUN apt-get install -y curl unzip build-essential openjdk-8-jdk-headless gradle
+RUN apt-get install -y curl unzip build-essential openjdk-8-jdk-headless wget
```

Comment:
Do we really need to maintain the builder container? Did you have a look at this: https://hub.docker.com/_/gradle?tab=description  ?

It is fine for the PR, but if there is no concrete reason we could (and probably should) use the standard Gradle image.

Reply:
I had tried that without it last week, and it did not work straight away, so I believe we need it but I am also not the expert on the topic.

Reply:
@oguzcankirmemis  What was the problem?

Reply:
Some package was simply missing (it was using the cache at this print), but I did not get a look on which it was, the error message was too long, so I had to force docker build without cache.

Reply:
This is a known issue. If the update line is cached, the new package lists are not fetched, and therefore packages are not found.
My question is rather about whether we need the (custom) container at all.

Reply:
I needed a very recent gradle version to work with the most recent Springframework plugin. With the default gradle from ubuntu image it did not build properly. That is why I decided to add the gradle version.